libcontainer/specconv/example.go

```diff
@@ -107,6 +107,12 @@ func Example() *specs.Spec {
 				Source:      "cgroup",
 				Options:     []string{"nosuid", "noexec", "nodev", "relatime", "ro"},
 			},
+			{
+				Destination: "/srv",
+				Type:        "bind",
+				Source:      "/srv",
+				Options:     []string{"bind", "ro"},
```

Comment:
`Example()` is used by quite a few projects, I'd prefer to not bind-mount host paths into containers by default.

Reply:
deleted.